.coveralls.yml

```diff
@@ -0,0 +1,4 @@
+service_name: travis-pro
```

Comment:
`travis-pro` is to private TravisCI, in this case is `travis-ci` for public repository